src/parser/PropertyPath.h

```diff
@@ -35,17 +35,12 @@ class PropertyPath {
                std::initializer_list<PropertyPath> children);
 
   static PropertyPath fromIri(std::string iri) {
+    AD_CONTRACT_CHECK(!iri.starts_with("?"));
```

Comment:
At least add a `TODO` to use a proper strongly typed IRI here in the future, and check, that the `iri` starts with "<".

Reply:
I will change this to a properly typed Iri in the future. I originally had the `<` check there, but it turned out to be to strict (I assume for language tagged iris there are some slightly different requirements)